scipy/integrate/_quadpack_py.py

```diff
@@ -866,6 +929,90 @@ def nquad(func, ranges, args=None, opts=None, full_output=False):
     fixed_quad : fixed-order Gaussian quadrature
     quadrature : adaptive Gaussian quadrature
 
+    **Details of QUADPACK level routines**
```

Comment:
This should solve the doc issue
```suggestion

	Notes
	-----

    **Details of QUADPACK level routines**
```

Reply:
For doc only changes, I suggest you to add this in your commit message (`[skip actions] [skip azp]`). This will only run circleCI